src/packages/quickenter/quickenter.scss

```
@@ -0,0 +1,122 @@
+@import '../popup/popup.scss';
```

Stylelint (17.11.0)

[error] Unexpected extension ".scss" in `@import` (scss/load-partial-extension)

(scss/load-partial-extension)